Blog/create token (#203)
StrataFoundation/strata: packages/docs/blog/2022-04-12-create-a-token/index.mdx

---
slug: create-a-token

> title: How to create a token on Solana

Reviewer: ```suggestion
title: Launch Solana Tokens on Strata a how to guide
```

> description: Launch Solana SPL Tokens using Strata Launchpad

Reviewer: ```suggestion
description: Learn how to launch Solana Tokens using the Strata Protocol Launchpad. 
```

tags: [guides, launchpad]
authors: [noah]
image: ./logo.png

> keywords: [how to, launch, create, spl token, solana, token, launchpad, permissionless, open source, social token, gamefi token]

Reviewer: ```suggestion
keywords: [solana tokens, cryptocurrency, launch token, spl token, solana defi, solana, launchpad, permissionless, open source, social token, gamefi token]
```

1. Creating the token. How do you actually create a token that will show up in wallets like Phantom, Slope, and Solflare?
2. Distribution. How do you get this token out to the community? How do you sell an initial supply to your community?
3. Liquidity. How does your community continually have the ability to buy and sell this token?

## Easy Mode - Fully Managed Tokens

Let’s say you do not want to worry about steps 1 through 3. You just want a token that people can buy and sell *right now*. When users put SOL in, they get the token out. You will not manage the token supply or pricing, and will instead take a royalty on sales (similar to an NFT). If you would like to create a token where you manage your own supply, you can skip to [Creating a Token](#creating-the-token)

For this article, I’m going to create a NOAH coin. Since it’s a small token, there’s no guarantee that there will be a counterparty to any trades. Instead, by using a Strata Fully Managed token, the protocol will be the counterparty to every trade.

Head over to the [Strata Launchpad](app.strataprotocol.com). Select “Create a Token”

> ![Create a Token](./create-a-token.png)

Reviewer: ```suggestion
![Create Solana Token](./create-a-token.png)
```

Select Fully Managed:

> ![Fully Managed](./fully-managed.png)

Reviewer: ```suggestion
![Fully Managed Solana Token](./fully-managed.png)
```

If this token isn't associated with a person, project, dao, it's fine to leave this toggle off.

Since I have set the royalties to 5% NOAH on Buy, I will get 5% of every NOAH token sale. If there are 100 NOAH tokens minted via the protocol, I will have received 5. I can also take SOL on buy/sell.

![Fully managed royalties/info](./social-token-info.png)

Done! You now have a tradable token! You can see this token on devnet [here](https://app.strataprotocol.com/swap/A6EpBzLEPw2CNrz7VWnWJUDLvjMx1KujtWxHYPjJk54m?cluster=devnet).

![Fully managed swap form](./fully-managed-done.png)

You can skip the rest of this guide, as it only applies to tokens not having their liquidity managed by Strata.

## Self Managed Tokens

A self managed token is one where you decide the supply, and have full control over distribution. You will want to think
about who should have this token, how it gets distributed, and the percentages/amounts for each party. You can see an example of this kind of workup [here](https://www.projectserum.com/serum-token-summary)

### Creating the Token

This is the easiest part. Head over to the [Strata Launchpad](app.strataprotocol.com) and select Create a Token

> ![Create a token](./create-a-token.png)

Reviewer: ```suggestion
![Create a Solana Token](./create-a-token.png)
```

Select self managed:

![Self managed](./self-managed.png)

Fill out your token’s information like the name, symbol, image, and supply:

![Self managed info](./fully-managed-info.png)

 Click Create Token. Strata will create the token and you should be able to see it in your wallet.

### Distribution

You have a token in your wallet, but how do you distribute it? You can send the token to people using your wallet, but what if you want to sell the token? This is called “Liquidity Bootstrapping.” In the previous step, you will have been greeted with a screen like this. If not, you can get to this screen by clicking “Sell a Token” on the launchpad.

> ![Sell a self managed token](./self-managed-done.png)

Reviewer: ```suggestion
![Sell a Solana Token](./self-managed-done.png)
```
